pkg/keys/sql.go

```diff
@@ -163,13 +163,13 @@ func (e sqlEncoder) ZoneKey(id uint32) roachpb.Key {
 	return MakeFamilyKey(k, uint32(ZonesTableConfigColumnID))
 }
 
-// MigrationKeyPrefix returns the key prefix to store all migration details.
+// MigrationKeyPrefix returns the key prefix to store all upgrade details.
 func (e sqlEncoder) MigrationKeyPrefix() roachpb.Key {
 	return append(e.TenantPrefix(), MigrationPrefix...)
 }
 
 // MigrationLeaseKey returns the key that nodes must take a lease on in order to
-// run system migrations on the cluster.
+// run system upgrades on the cluster.
 func (e sqlEncoder) MigrationLeaseKey() roachpb.Key {
```

Comment:
s/Migration/Upgrade?

Reply:
Turns out these are actually about `startupmigration`s which I am avoiding because I want them to go away https://github.com/cockroachdb/cockroach/issues/73813. I made that more explicit.